src/models/Actor.py

```diff
@@ -0,0 +1,21 @@
+from src.models.Person import Person
+
+
+class Actor(Person):
+    """
+    This class inherits of the Person class. Actor's objects have a picture (pict).
+    """
+    def __init__(self, name: str, surname: str, pict: str):
+        Person.__init__(self, name, surname)
+        self.__pict = pict
+
+    @property
+    def pict(self):
+        return self.__pict
+
+    @pict.setter
+    def pict(self, pict):
+        if type(pict) != str:
```

Comment:
Après réflexion on s'est dit qu'on aller mettre pour les vérifications de type `if type(obj) is not class :` plutôt que `if type(object) != cls`